edelweiss_data/api.py

```diff
@@ -752,6 +753,7 @@ def decode(cls, d, api):
             description=d['description'] if 'description' in d else None,
             metadata=d['metadata'] if 'metadata' in d else None,
             rowcount=d['rowcount'] if 'rowcount' in d else None,
+            is_public=d['isPublic'] if 'isPublic' in d else None,
```

Comment:
Why don't we just use `is_public=d.get('isPublic')` here and in the lines above? I seem to recall I wrote this, but it was originally only for stuff such as `Schema.decode(d['schema']) if 'schema' in d else None` so that you don't call `decode` on `None`.

Reply:
I was wondering exactly the same thing as I added this line.  I will change all occurrences.

Reply:
:+1: